tests/sles4sap/qesapdeployment/test_cluster.pm

```diff
@@ -24,13 +24,25 @@ sub run {
     record_info("crm status", $cmr_status);
     qesap_ansible_cmd(cmd => $crm_mon_cmd, provider => $prov, filter => '"hana[0]"');
     qesap_cluster_logs();
+
+    if (get_var("QESAP_IBSMIRROR_RESOURCE_GROUP")) {
+        my $rg = qesap_get_az_resource_group();
+        my $ibs_mirror_rg = get_var('QESAP_IBSMIRROR_RESOURCE_GROUP');
+        qesap_az_vnet_peering(source_group => $rg, target_group => $ibs_mirror_rg);
+        qesap_delete_az_peering(source_group => $rg, target_group => $ibs_mirror_rg);
+    }
```

Comment:
This code (L28-L33) can go into a function inside the test module, and then you call the function here, and in the `post_fail_hook()` as it's the same.

Reply:
I can think about moving 28-31. In other test modules,  delete is not exactly after the create.